### PR TITLE
Add task to Sandbox Data Pipeline to Trigger Anomalo Checks on Snowflake Cocktails Staging Table

### PR DESCRIPTION
**Pull request by njfritter**
The Sandbox Data Pipeline was currently missing a task to trigger Anomalo checks on the Snowflake Cocktails staging table because it was not showing up in Qbiz's Anomalo account.

Now that the above issue was solved via #25, that task should be added to the DAG.

### CODE REVIEW

File: dags/sandbox_data_pipeline.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 from airflow.operators.empty import EmptyOperator
 from airflow.providers.google.cloud.operators.bigquery import BigQueryInsertJobOperator
 
-sys.path.append("/opt/airflow")
+sys.path.append("/home/airflow/airflow")
```
Comment (sorenarchibald) [inline dags/sandbox_data_pipeline.py:20]: Let's not hardcode this path.  I might want to switch it back to /opt/airflow when we move to ECS.  Is AIRFLOW_HOME the value we want?

**Review by sorenarchibald — CHANGES_REQUESTED** [review]
`os.env("AIRFLOW_HOME")` Maybe?  see comments

File: dags/sandbox_data_pipeline.py (hunk 1 above)
Comment (njfritter) [reply dags/sandbox_data_pipeline.py:20]: Yes you are correct:

Screenshot 2024-04-11 at 5 21 40 PM


I'll go and make that change.

Comment (njfritter) [reply dags/sandbox_data_pipeline.py]: @sorenarchibald Check now when ready

**Review by sorenarchibald — APPROVED** [review]

**Pull request merged**